src/libs/actions/Session/index.js

```diff
@@ -248,6 +257,53 @@ function signIn(password, validateCode, twoFactorAuthCode) {
     API.write('SigninUser', params, {optimisticData, successData, failureData});
 }
 
+function signInFromMagicLink(accountID, validateCode) {
```

Comment:
We try to avoid using marketing terms like "magic code" and "magic link" in the code base, so I think `signInWithValidateCode` might be better

Reply:
Also accountID appears to be unused in this function

Reply:
> `signInWithValidateCode` might be better

I like it, thanks for suggesting!

> Also accountID appears to be unused in this function

We should actually authenticate based on the accountID and validateCode but auth doesn't support it yet and we can't make it yet given [this](https://expensify.slack.com/archives/C048ZLJ78P5/p1675200659470319) Slack discussion. But yes, I do agree that I shouldn't let unused variable based on assumptions that they would eventually be used in the future.

Fixed both of the above.